app/dashboard/views.py

```diff
@@ -2084,3 +2092,38 @@ def get_kudos(request):
         raise Http404
     mimetype = 'application/json'
     return HttpResponse(data, mimetype)
+
+
+def hackathon(request, hackathon=''):
+    """Handle rendering of HackathonEvents. Reuses the dashboard template."""
+
+    evt = None
+    try:
+        evt = HackathonEvent.objects.filter(slug__iexact=hackathon).latest('id')
+        title = evt.name
+    except HackathonEvent.DoesNotExist:
+        raise Http404
+
+    params = {
+        'active': 'dashboard',
+        'title': title,
+        'keywords': json.dumps([str(key) for key in Keyword.objects.all().values_list('keyword', flat=True)]),
+        'hackathon': evt,
+    }
+    return TemplateResponse(request, 'dashboard/index.html', params)
+
+
+def get_hackathons(request):
+    """Handle rendering all Hackathons."""
+    evt = None
```

Comment:
Can we name one of these variables "event" and the other "events" or something similar to make this code a bit more readable?